ci/release-container/Dockerfile

```diff
@@ -26,18 +27,18 @@ ENV SHELL=/bin/bash
 RUN adduser --gecos '' --disabled-password coder && \
   echo "coder ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/nopasswd
 
-RUN curl -SsL https://github.com/boxboat/fixuid/releases/download/v0.4/fixuid-0.4-linux-amd64.tar.gz | tar -C /usr/local/bin -xzf - && \
+ARG ARCH=amd64
```

Comment:
This doesn't really matter but since I'm thinking about it what are your thoughts on this vs the `uname` + `sed` method of getting the arch used in the CI container?

Reply:
uname + sed sounds better.

Reply:
Though tbh it's nice right now to not clobber the dockerfile. Using uname + sed will make that line even harder to fall.

Reply:
Yeah I don't feel super strongly about it, it's mostly this odd feeling I get since technically you can build a broken Docker image if you provide the wrong arg (or no arg). 

But even in the theoretical world where the arg was wrong somehow that would mean the tag would also be wrong so it'd be broken in practice for users anyway. 

Which means the only place this could reasonably come into play is if someone was trying to build it on their own or something.

Reply:
Fair, let's copy lib.sh in and source it and then just run the arch function.

Reply:
Dope, that sounds perfect.